ocean_lib/models/data_token.py

```diff
@@ -217,17 +217,18 @@ def verify_order_tx(self, web3, tx_id, did, service_id, amount_base, sender):
 
         # verify sender of the tx using the Tx record
         tx = web3.eth.getTransaction(tx_id)
-        if tx['from'] != sender:
+        if order_log.args.consumer != sender and order_log.args.payer != sender:
```

Comment:
to avoid and/or ambiguity, can we make this condition: `if sender not in [order_log.args.consumer, order_log.args.payer]`?